doc/api/http.md

```diff
@@ -1401,9 +1398,14 @@ Limits maximum incoming headers count. If set to 0, no limit will be applied.
 
 <!-- YAML
 added: v14.11.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/41263
+    description: The default request timeout changed
+                 from no timeout to 300s (5 minutes).
 -->
 
-* {number} **Default:** `0`
+* {number} **Default:** `300000`
```

Comment:
Hey! I believe this change should be added to `https.md` as well :slightly_smiling_face:

Reply:
@julien-f Yes, it should. Are you willing to send a PR about this or will I do that?

Reply:
Please do it, I don't have the time at the moment.

Thank you :slightly_smiling_face:

Reply:
Ok!